requirements.txt

```diff
@@ -1,3 +1,4 @@
 mock==2.0.0
 nose==1.3.7
 tox==2.6.0
+codecov
```

Comment:
please pin this to a specific version, that way the build won't fail out of band if this dependency is upgraded and becomes incompatible for some reason.
```
pip freeze | grep codecov
codecov==2.0.9
```